lib/network/modules/SelectionHandler.js

```diff
@@ -601,6 +601,44 @@ class SelectionHandler {
     return idArray;
   }
 
+  /**
+   * Updates the current selection
+   * @param {{nodes: Array.<String>, edges: Array.<String>}} Selection
+   * @param {Object} options                                 Options
+   */
+  setSelection(selection, options = {}) {
+    let i, id;
+
+    if (!selection || !selection.nodes || !selection.edges)
```

Comment:
For convenience, we could allow only providing one of the two: `nodes` or `edges`, i.e. allow `Network.setSelection({nodes: [1,2,3]})`.